File: python/programmers/2016.py
a = int(input("월입력: "))
b = int(input("일입력: "))

def solution(a, b):
	month = [0, 31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
	week = ["THU", "FRI", "SAT", "SUN", "MON", "TUE", "WED"]
	'''
	a = sum(month[:a])+b
	b = a%7
	return week[b]
	'''
	return week[(sum(month[:a])+b)%7]


print(solution(a, b))

File: c/programmers/level1/2016.c
#include <stdio.h>
#include <stdlib.h>

char* solution(int a, int b)
{
	int i;
	int sum = 0;
	int ret;
	char* answer = (char*)malloc(sizeof(char)*4);
	int month[13] = {0, 31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31};
	char* week[7] = {"THU", "FRI", "SAT", "SUN", "MON", "TUE", "WED"};
	for(i=0; i<a; i++)
	{
		sum += month[i];
	}
	sum += b;
	ret = sum%7;
	answer = week[ret];

	return answer;
}

int main(void)
{
	int a, b;
	printf("월 입력: ");
	scanf("%d", &a);
	printf("일 입력: ");
	scanf("%d", &b);
	printf("%d월 %d일 의 요일은: %s \n", a, b, solution(a, b));
	return 0;
}
